### DISCUSSION TITLE
Having a little crack at trying to workflow into Raspberry Noaa V2

### DISCUSSION THREAD

**Discussion by dom-robinson** [martinber/noaa-apt#41] [General]
Hey Martin - some while ago i was working a lot on [https://github.com/jekhokie/raspberry-noaa-v2](https://github.com/jekhokie/raspberry-noaa-v2) and partially got noaa-apt integrated in an early release, but [as you will see](https://github.com/jekhokie/raspberry-noaa-v2/pull/440), ran out of time and energy to get it finished and PR it formally. 

I have just brought my efforts forward to the current release of RN2 over lunch today since the main master of RN2 has slowed pace a lot now (the main dev got sidetracked and left a few of us with it) but until its tested and done 'right' at deployment (its a manual ```cp``` at the moment (!) ) its not quite right - but i thought i'd tag you here since i am unlikely to get to it again for a while...

I like NOAA-APT - nice project! Thank you! 

(Be nice to have the time to finish these things properly one day!!! )

Dom

**Comment by martinber** [comment]
Hello,

So if I understand, part of the problem is how noaa-apt searches for the map overlays in the `res/` folder right? I thought that it made sense to search in the current working directory as done now, but I'm starting to think that I'm wrong since you are not the first to have that problem haha.

So, what do programs normally do when they have to search for files? Right now it searches for `/usr/share/noaa-apt/res/` if installed via .deb and `./res/` otherwise. This is selected via the compile-time flag `NOAA_APT_RES_DIR`. Should I add a command-line argument as requested here #37 ?
